.github/workflows/ci.yml

```diff
@@ -101,6 +102,7 @@ jobs:
           $env:Path += ";" + "$($PWD)\ManimCache\SoX"
           $env:Path = "$env:USERPROFILE\.poetry\bin;$($env:PATH)"
           echo "::set-env name=Path::$env:Path"
+          pip install pyreadline
```

Comment:
Do you know which pip is used here to install `pyreadline`? If it is not the one from the virtualenv provided by poetry, this might not make a lot of sense, or do I miss something?

It does not seem like it is used anywhere in the current code base (and I am also not sure whether the import was actually used before the removal of the live streaming feature in the PR you linked somewhere). Maybe we can simply remove it from our dependencies?

(CC @huguesdevimeux, not sure whether it `pyreadline` is required for #441 ...)

Reply:
Yeah, it should be removed. I will soon remove it. 
About poetry virtual environment, I had a environment variable set so that it doesn't create a new virtual environment instead install globally. 
So pretty much installing directly globally using pip will do as in this PR. Once, I remove the dependency of pyreadline this PR can be merged. But wait until I do so.